cmd/ova-provider-server/ova-provider-server.go

```diff
@@ -429,6 +431,45 @@ func readOVF(ovfFile string) (*Envelope, error) {
 	return &envelope, nil
 }
 
+const (
+	Unknown    = "Unknown"
+	VMware     = "VMware"
+	VirtualBox = "VirtualBox"
+	RHV        = "RHV"
+	Xen        = "Xen"
+)
+
+// Check the OVF XML for any markers that might cause import problems later on.
+// Not guaranteed to correctly guess the OVA source, but should be good enough
+// to filter out some obvious problem cases.
+func guessOvaSource(envelope Envelope) string {
+	namespaceMap := map[string]string{
+		"http://schemas.citrix.com/ovf/envelope/1": Xen,
+		"http://www.citrix.com/xenclient/ovf/1":    Xen,
+		"http://www.virtualbox.org/ovf/machine":    VirtualBox,
```

Comment:
You are missing the ovirt, here is ovf example 
[vm.ovf.txt](https://github.com/user-attachments/files/20069993/vm.ovf.txt)

Reply:
Thanks, added.